test/NugetFeedDeploymentRepositoryTest/NugetFeedApplicationRepositoryTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Etg.Yams.Application;
+using Etg.Yams.NuGet.Storage;
+using Etg.Yams.Storage;
+using Etg.Yams.Utils;
+using Xunit;
+
+namespace Etg.Yams.Nuget.Storage.Test
+{
+    [Trait("Category", "Integration")]
```

Comment:
This doesn't run in CI. I'd remove the *integration* trait unless there is a reason why the tests should not run as part of the CI build.

Reply:
Done. Didn't know _integration_ means no CI execution.